pages/common/bird.md

```diff
@@ -0,0 +1,13 @@
+# bird
+
+> BIRD Internet Routing Daemon.
+> Routing daemon with support for BGP, OSPF, Babel and others.
+> More information: <https://bird.network.cz/>.
+
+- Start Bird with a specific configuration file:
+
+`bird -c {{path/to/bird.conf}}`
+
+- Start Bird as a specific user and group:
+
+`bird -u {{bird}} -g {{bird}}`
```

Comment:
```suggestion
`bird -u {{username}} -g {{group}}`
```
quite unclear placeholders just using bird, or is bird:bird just the most commonly used user:group combination?

Reply:
These are the default user and group generated when installing on Debian, but yes here it is unclear.